ios/Surveys/Sources/Presentation/Modules/SurveyQuestions/SurveyQuestionsView.swift

```diff
@@ -0,0 +1,141 @@
+//  swiftlint:disable function_body_length
+//  SurveyQuestionsView.swift
+//  Surveys
+//
+//  Created by Su Ho on 27/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+import SwiftUIPager
+
+struct SurveyQuestionsView: View {
+
+    @ObservedObject private var viewModel: SurveyQuestionsViewModel
+    @EnvironmentObject private var navigator: Navigator
+    @StateObject private var page: Page = .first()
+
+    var body: some View {
+        switch viewModel.state {
+        case .idle:
+            content().onAppear { viewModel.loadData() }
+        case .loading:
+            content(isLoading: true)
+        case .loaded:
+            content()
+        case let .failure(message):
+            content().defaultAlert(message: message)
+        }
+    }
+
+    var background: some View {
+        ZStack {
+            GeometryReader { proxy in
+                URLImage.urlString(viewModel.imageURLString)
```

Comment:
I think we should handle the survey's URL image in each page because each question has its own background image :D.

Reply:
Based on the design, it only uses 1 image, so I prefer to keep using this one :D